README.md

```diff
@@ -134,7 +136,7 @@ And add Canvas to your dependencies
 
 ```gradle
 dependencies {
-	modCompile "grondag:canvas-mc116:1.0.+"
+	modCompileOnly "grondag:canvas-mc116:1.0.+"
```

Comment:
Should probably be `grondag:canvas-mc117-1.17:1.0.+` and throw in `modRuntime` as well for a good measure (need Grondag to confirm this).

Reply:
Probably  yes.